drivers/ThinQ_Washer.groovy

```diff
@@ -124,20 +124,26 @@ def processStateData(data) {
     }
     if (data?.containsKey('Remain_Time_M') ) {
       remainingTime += (data["Remain_Time_M"]*60)
+    } else {
+      remainingTime = 0
```

Comment:
I don't think this is needed. If Hours is set but not Minutes, shouldn't we just display the hours value, not 0?